src/components/InstancePlugin/InstancePlugin.test.js

```diff
@@ -0,0 +1,25 @@
+import '../../../test/jest/__mock__';
+import { screen } from '@folio/jest-config-stripes/testing-library/react';
+import InstancePlugin from './InstancePlugin';
+import { renderWithIntl } from '../../../test/jest/helpers';
+
+const onSelectMock = jest.fn();
+
+describe('InstancePlugin', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should InstancePlugin', () => {
+    renderWithIntl(<InstancePlugin onSelectMock={onSelectMock} isDisabled={false} />);
+
+    expect(screen.getByText(/findInstancePluginNotFound/i)).toBeInTheDocument();
+  });
+
+  it('should InstancePlugin with is disabled prop', () => {
+    renderWithIntl(<InstancePlugin onSelectMock={onSelectMock} isDisabled />);
+
+    expect(screen.getByText(/findInstancePluginNotFound/i)).toBeInTheDocument();
+  });
+});
```

Comment:
I think your second test is redundant.
You do not check any enabled and/or disabled elements on the page after component rendering. You just pass different properties that do not have impact on any elements.

Reply:
Hi @artem-blazhko, agree, I've removed

cc @UladzislauKutarkin